### PR TITLE
Chinese translation of PHP 8.1 Announcement page

### PR DESCRIPTION
**Pull request by sy-records**
done, cc @saundefined @cmb69 @salathe @nikic

### CODE REVIEW

**Comment by sy-records** [comment]
@shiny @jhdxr 可以帮忙看看翻译有什么问题吗

File: releases/8.1/languages/zh.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	'other_new_returntypewillchange' => '<code>#[ReturnTypeWillChange]</code> 属性。',
+	'other_new_fsync_fdatasync' => '<code>fsync</code> 和 <code>fdatasync</code> 函数。',
+	'other_new_array_is_list' => '<code>array_is_list</code> 函数。',
+	'other_new_sodium_xchacha20' => 'Sodium XChaCha20 函数。',
+
+	'bc_title' => '弃用和向后不兼容',
+	'bc_null_to_not_nullable' => '向非空值的内部函数参数传递空值的做法已被弃用。',
+	'bc_return_types' => 'PHP 内置类方法中的暂定返回类型',
+	'bc_serializable_deprecated' => '<code>Serializable</code> 接口已弃用。',
+	'bc_html_entity_encode_decode' => 'HTML 实体 en/decode 函数默认处理单引号和替换。',
```
Comment (shiny) [inline releases/8.1/languages/zh.php:73]: `HTML entity en/decode functions process single quotes and substitute by default.`
这里的替换有点不明白意思，看了下 `html_entity_decode` 的 changelog，`flags changed from ENT_COMPAT to ENT_QUOTES \| ENT_SUBSTITUTE \| ENT_HTML401.` 
估计这里的替换是指 ENT_SUBSTITUTE，用 Unicode 替换字符来替换无效字符
单引号是指以前 ENT_COMPAT 不会处理单引号，现在 ENT_QUOTES 也会处理单引号了。
可以在尊重英文原文的基础上有更易理解的补充。

File: releases/8.1/languages/zh.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	'bc_pspell_objects' => 'Pspell：pspell 和 pspell config 资源类型迁移到 <code>PSpell\Dictionary</code>、<code>PSpell\Config</code> 类对象。',
+
+	'footer_title' => '更好的性能、更好的语法、改进类型安全。',
+	'footer_content' => '<p>
+        请访问 <a href="http://www.php.net/downloads">下载</a> 页面下载 PHP 8.1 源代码。
+        在 <a href="http://windows.php.net/download">PHP for Windows</a> 站点中可找到 Windows 二进制文件。
+        <a href="https://www.php.net/ChangeLog-8.php#PHP_8_1">ChangeLog</a> 中有变更历史记录清单。
+      </p>
+      <p>
+        PHP 手册中有 <a href="/manual/en/migration81.php">迁移指南</a>。
```
Comment (shiny) [inline releases/8.1/languages/zh.php:92]: 这里的链接可以替换为中文版的 `/manual/zh/migration81.php`

**Review by shiny — COMMENTED** [review]
简单看了下，没有啥大问题，挑了一些刺，供参考 😂

**Comment by saundefined** [comment]
@sy-records is it ready to merge?

It seems like we need to improve the layout of the "新的类、接口和函数" heading a bit.
If the translation is ready, I would merge the changes along with the layout improvements.

<details>
  <summary>example</summary>
  
Снимок экрана 2021-11-27 в 17 58 49
</details>

**Comment by sy-records** [comment]
> @sy-records is it ready to merge?

yep.

**Review by saundefined — APPROVED** [review]
Thank you!

**Pull request merged**